evm_arithmetization/tests/erc721.rs

```diff
@@ -52,6 +52,7 @@ type C = KeccakGoldilocksConfig;
 /// `1337` from address `0x5B38Da6a701c568545dCfcB03FcB875f56beddC4` to address
 /// `0xAb8483F64d9C6d1EcF9b849Ae677dD3315835cb2`.
 #[test]
+#[ignore]
```

Comment:
Is erc721 taking that long that we're ignoring it on default?

Reply:
Yes, I found that the test running on CI is about 4x slower than on my MacBook. The test time in release mode on my MacBook can be found here: https://github.com/0xPolygonZero/zk_evm/pull/739

Reply:
If we're going to start doing test selection like this, I think we need a better test runner (`cargo-nextest`)

It allows you to categorise your tests, have different timeouts, profiles for ci etc